src/checkout-buttons/create-checkout-button-registry.ts

```diff
@@ -6,7 +6,7 @@ import { CheckoutActionCreator, CheckoutRequestSender, CheckoutStore } from '../
 import { Registry } from '../common/registry';
 import { ConfigActionCreator, ConfigRequestSender } from '../config';
 import { BraintreeScriptLoader, BraintreeSDKCreator } from '../payment/strategies/braintree';
-import { createGooglePayPaymentProcessor, GooglePayBraintreeInitializer, GooglePayStripeInitializer } from '../payment/strategies/googlepay';
+import { createGooglePayPaymentProcessor, GooglePayBraintreeInitializer, GooglePayCheckoutcomInitializer, GooglePayStripeInitializer } from '../payment/strategies/googlepay';
```

Comment:
This line is very long, could you please separate it into several lines? My example might not be correctly indented 
```suggestion
import { 
    createGooglePayPaymentProcessor, 
    GooglePayBraintreeInitializer,
    GooglePayCheckoutcomInitializer,
    GooglePayStripeInitializer
} from '../payment/strategies/googlepay';
```

Reply:
@FranciscoOnt it's not possible due to:
https://github.com/bigcommerce/checkout-sdk-js/blob/6931d3e46f1e3e47abb36dc0c1b3f23d09b85fab/.eslintrc.json#L45

Reply:
can't be done